lib/sdl.cpp

```diff
@@ -652,6 +652,7 @@ void SdlWindow::mainIter()
       {
          if (glvis_command->Execute() < 0)
          {
+            cout << "GlvisCommand signalled exit" << endl;
```

Comment:
Styling should be `GLVisCommand` or `glvis_command`

Reply:
Not actually sure how this addition made it in, so I just removed it.